Community-Projects.md

```diff
@@ -41,5 +41,6 @@ _Server implementations related to enhancing building._
 * [plots](https://github.com/df-mc/plots) - Minecraft plots server written using Dragonfly.
 
 ## Tools
+* [strutils](https://github.com/unickorn/strutils) - Library for text centering and color code cleaning.
```

Comment:
I would suggest maybe moving it under the `Libraries` into a new `Text Formatting` section. I forgot to add text here, but `Tools` is intended for stand-alone (executable) tools. Would that work?

Reply:
Sure!